cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-configmaps-secrets-env-var.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-configmaps-secrets-envvar"]
+= Tutorial: ConfigMaps, secrets, and environment variables
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-configmaps-secrets-envvar
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 05-07-2024
+
+This tutorial shows how to configure the OSToy application by using xref:../../nodes/pods/nodes-pods-configmaps.adoc[config maps], link:https://docs.openshift.com/container-platform/latest/cicd/builds/creating-build-inputs.html#builds-input-secrets-configmaps_creating-build-inputs[secrets], and link:https://docs.openshift.com/container-platform/3.11/dev_guide/environment_variables.html[environment variables]. See these linked topics for more information.
```

Comment:
Give the "why" before the "how". So rewrite as: "For more information, see these linked topics."